screens/login/index.js

```diff
@@ -2,10 +2,9 @@
 
 import Login from './login-screen';
 import ForgotPassword from './forgot-password';
+import { createStackNavigator } from 'react-navigation';
 
-import {StackNavigator} from 'react-navigation';
-
-export default StackNavigator({
+export default createStackNavigator({
```

Comment:
The warning message on the login screen was a simple fix: `StackNavigator` had been deprecated in favor of `createStackNavigator`.